### DISCUSSION TITLE
似乎是bug

### DISCUSSION THREAD

**Discussion by AMDGOOD** [husandro/WoWTools#8] [Q&A]
拾取记录队友的记录会显示两次名字。
![QQ截图20221218145754](https://user-images.githubusercontent.com/74037851/208285666-2fbf835c-b28f-41a3-8900-88185f306307.jpg)
无法取消小退，倒数框点掉后依然在小退倒数且不显示，角色不能动。

**Comment by husandro** [comment]
> 拾取记录队友的记录会显示两次名字。
> ![QQ截图20221218145754](https://user-images.githubusercontent.com/74037851/208285666-2fbf835c-b28f-41a3-8900-88185f306307.jpg)
> 无法取消小退，倒数框点掉后依然在小退倒数且不显示，角色不能动。
> 

可能是聊天时间开启有这问题, 
试试关闭,系统聊天时间, 

我试试解决这个问题,
